### DISCUSSION TITLE
HAOS for Khadas vim1s

### DISCUSSION THREAD

**Discussion by giallu** [home-assistant/operating-system#3712] [General]
I've noticed there is a Khadas vim3 image in the list of release artifacts, so what would it take to also add a vim1s image?

is it possible to try building it locally? what would the process look like?

**Comment by sairon** [comment]
VIM1S uses an entirely different SoC, and since VIM3's usage is marginal, the effort for adding support for and maintaining this board is hardly justifiable.

Of course you could run a custom build on it, but it's hard to estimate the effort and there would be definitely some learning curve if you're not familiar with (embedded) Linux development. At first it would mean a new Buildroot config with different device tree and possibly Linux defconfig must be added where VIM3's one could probably serve as the starting point. What's next depends on the result of that effort.

**Comment by jerome-diver** [comment]
There is an image on Khadas forum that you can find to get running HA supervised on Armbian OS Bookworm.
It is the best choice at this date that you can get to run HA with ability to use driver. The limit is kernel 6 (they are on the process to build a support for that). 

Khadas Vim1s is a very nice hardware because he has exactly what HA (or any little server) asking as power to be run, but the best point is his low electric consumed, at an accessible price.

So i totally agree with you, HA OS should consider and target on Vim1s device and not just follow the "last device to be" around the "SBC tourism world".

Also, they (HA OS dev team) maybe can consider to not move all the time there files in there project or change directories names so many times because it will be difficult to get trustable information after that move/change. 

The maintainability is a crucial point for devs on a project to go with.

**Comment by lockheed** [comment]
Since HA team announced dropping support for HA Supervised, it would be good to have two things:
1. A guide on how to migrate from HA Supervised to HA Docker install, since non-manual backup restore is not possible here.
2. A HAOS image for Vim1s, as there is one for Vim3.

**Reply by PedroC44** [reply]
Seems there is a new  image for haos/vim1s here : https://dl.khadas.com/.images/vim1s/

**Reply by tesKingIT** [reply]
Hi, how can I use this? I'm a Newbie...

**Reply by PedroC44** [reply]
i'm a newbie too😀. But https://docs.khadas.com/software/oowow/how-to/start can help you. The only installation I did was via oowow and the image was in the list displayed by oowow.

**Reply by lockheed** [reply]
Does someone know what are the differences to the old image? I have a working setup and am not quite happy to sacrifice several hours to find out.

**Reply by giallu** [reply]
The new image is HAOS (which is what I was hoping to find and/or create here) while the old one has supervised, which was deprecated by the guys at HA.

**Reply by lockheed** [reply]
Great info, thanks. Did they change the partition layout/table, or is it still non-standard and unreadable/unservicable outside the OS itself, as it was on the old image? What file format is being used and on what partition layout/sizes?

**Reply by tesKingIT** [reply]
> i'm a newbie too😀. But https://docs.khadas.com/software/oowow/how-to/start can help you. The only installation I did was via oowow and the image was in the list displayed by oowow.

Thank you so much! I follow the guide section "Write custom OS image into eMMC from removable storage", putting the image on a USB dongle. It worked! 3 mins to push the image on eMMC, 20 mins to first bootup and self setup; at the end I restored my last backup, other 20 mins... and now My system is HAOS and no more Supervised... Perfect!

Cristian - tesKing

**Comment by lockheed** [comment]
Does someone know what are the differences to the old image? I have a working setup and am not quite happy to sacrifice several hours to find out.

**Reply by jerome-diver** [reply]
1/ there is many differences between images that you have to track because of so many depend of the concern
2/ On open source and free software the price you paid is your time you are investing to search and find what other did for free. Sure, no one like to loose time and break something that is working.

My idea is to always hold what is working and go ahead only for some good reasons (but sure not the last one i;age to exist or a consideration of the order of aesthetics).
An other one idea can be to: 1/ manage a recover plan in can of problem, 2/ find the time need to investigate with all your requested technologies concerned by the future system (yes, choices can be different depend of your goal).

3/ differences about different possibilities to get a HA is well explained in HA web site (also, they change things a lot if you read there github project... it is a bit annoying but well... maybe it is better than nothing to happen) or now you can ask to perplexity.ai and his answer is correct on this subject (but not in precises points about latest things, for sure... it is the free plan).

**Reply by lockheed** [reply]
Thanks for that thoroughly useless answer. You brought nothing new or constructive to the conversation, but added some irrelevant stuff about HA changes, when the question was clearly about image changes, not HA. I'd rank your previous answer in this thread head and shoulders about that one - but that's not saying much.

**Reply by jerome-diver** [reply]
thank you to show me the perfect example of what you are speaking about. thank you to ask and for all your help around. nice.
